### ISSUE TITLE
Onion Skinning is not very intuitive

### ISSUE THREAD

**Issue by rioforce** [charlielee/boats-animator#71]
While this is mainly my issue, since I redesigned the entire Onion Skinning slider, I thought I'd bring it up so we can discuss it. Basically, as it is, the onion skinning slider isn't very easy to understand. It is in the middle, and you can drag it up and down to change the onion skinning factor, but it only takes effect if the Onion Skinning button is enabled. The button and slider act awkwardly together, and it's not really easy to understand what each is at first glance.

So, I am thinking we need to find a way to get rid of the button and use the slider exclusively. One way to do this is to make the onion slider be at the bottom of the slider by default, and when you drag it up and down, it adjusts the opacity. This removes the need for a separate onion skinning button, but it makes the slider look off-balance with everything, since the circle isn't in the middle.

Another way to do it is to make the left and right portions of the slider do different things. It works like this in Dragonframe:
- Drag to the right: Onion skins last frame taken with live preview. If you select another frame, it doesn't show any live preview or onion skinning.
- Drag to the left: Onion skins last frame taken with live preview if on live mode. If you select another frame in the reel, it'll use that for onion skinning instead.

We don't have to do it like this, I'm just unsure what we need to do. Eliminating a need for a separate button would help a lot.
What do you guys think?

**Comment by Robotic-Animations** [comment]
I agree that the button and slider is confusing, especially when using the program for the first time. I personally like the second proposal, as it seems like a more intuitive and versatile solution.

**Comment by BoatsAreRockable** [comment]
Working towards a solution where the slider does different things based on if it is dragged left or right seems good. Although, as I mentioned in your pull request, the playback code must be sorted out first so this can be accommodated. 

We could partially implement this by removing the toggle button and having the last frame onion skin when the slider is dragged to the right. For now, moving the slider left of centre will do nothing.

**Comment by BoatsAreRockable** [comment]
The keyframe system I'm currently working on in #116 could allow this to be implemented.

**Comment by BoatsAreRockable** [comment]
If my keyframe system in #122 is implemented this feature could work so that sliding to the left shows the current keyframe as the onion skinning frame and sliding to the right shows the last captured frame.

**Comment by le717** [comment]
If you think you can implement this once keyframes lands, then I'll go ahead and move this to 0.7.x.

**Comment by le717** [comment]
@Robotic-Animations @rioforce You'll be happy to know this is currently in development at #122. 😃

**Comment by charlielee** [comment]
I've created a new issue for enabling different behaviour when the slider is moved to the left. #320 Who knew I'd still be working on Boats Animator 6 years on! 😂
